### DISCUSSION TITLE
Uninstalling Wiki.js

### DISCUSSION THREAD

**Discussion by stathri** [requarks/wiki#5889] [Help / Questions] [answered]
How would I go about uninstalling Wiki.js? Wiki.js prevents my Nginx sites from being reachable so I'm probably going to have to get a new server just for Wiki.js.

**Comment by stathri** [comment]
And on that new server, how would I go about running two Wiki.js instances on two different subdomains?

**Comment by fnafox** [comment, marked as answer]
This isn't a wikijs related issue, but If your wiki prevents your other sites from being reachable then that's an issue with your virtualhost configuration in nginx. There is no reason why you couldn't run it on the same host. Running multiple instances of wikijs works on the same principle where you'd use a reverse proxy for each subdomain that points to the correct socket on your host.

I'm not too familiar with nginx so I can't help with the exact configuration, look up nginx virtualhost setup online.

On apache this would look something like this:
```
<VirtualHost *:80>
    ServerName wiki.example.com
    ProxyPreserveHost Off
    ProxyPass / http://localhost:3000/
    ProxyPassReverse / http://localhost:3000/
</VirtualHost>
<VirtualHost *:80>
    ServerName wiki2.example.com
    ProxyPreserveHost Off
    ProxyPass / http://localhost:3001/
    ProxyPassReverse / http://localhost:3001/
</VirtualHost>
```

**Comment by NGPixel** [comment]
As fnafox mentionned, this is not a Wiki.js issue. Your nginx configuration is wrong.

You could have 2 different Wiki.js instances running on port 3000 and 3001 respectively. Then simply point to each using a server block in your nginx configuration, e.g.:

```nginx
server {
    listen 80;
    server_name wiki.example.com;

    location / {
      proxy_pass http://127.0.0.1:3000;
    }
}
server {
    listen 80;
    server_name wiki2.example.com;

    location / {
      proxy_pass http://127.0.0.1:3001;
    }
}
```
